functions.d/ambient_get_icomera_endpoint.fish

```diff
@@ -0,0 +1,9 @@
+function ambient_get_icomera_endpoint
+	set url https://www.ombord.info/api/jsonp/$argv/
+
+	if test (uname) = "Darwin"
+		curl -s $url | tail -c +2 | ghead -c -3
```

Comment:
Thought: we could do something like this in `functions.d/head.fish`:

```
function head
  env (command -s head ghead)
end
```

I'm not entirely sure what the idiomatic equivalent of `$(command -s ...)` is in fish, but `env` works as a hack.

Reply:
<img width="1082" alt="Screenshot 2020-01-30 at 5 40 44 pm" src="https://user-images.githubusercontent.com/6197148/73475119-a66c9000-4387-11ea-81f5-c036b0be8ed3.png">
Doesn't seem to be working right for me

Reply:
I don't know enough about how to go about doing something like this, hence the `if`